### PR TITLE
BXMSDOC-7726-master: Adopt SME feedback

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/admin-and-config/manage-service-tasks-proc.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 = Managing custom tasks in {CENTRAL}
 
-Custom tasks (work items) are tasks that you can customize and reuse across multiple business processes or across all projects in {CENTRAL}. {PRODUCT} provides a set of custom tasks within the custom task repository in {CENTRAL}. You can enable or disable the default custom tasks and upload custom tasks into {CENTRAL} to implement the tasks in the relevant processes.
+Custom tasks (work items) are tasks that can run custom logic. You can customize and reuse custom tasks across multiple business processes or across all projects in {CENTRAL}. You can also add custom elements in the designer palette, including name, icon, sub-category, input and output parameters, and documentation. {PRODUCT} provides a set of custom tasks within the custom task repository in {CENTRAL}. You can enable or disable the default custom tasks and upload custom tasks into {CENTRAL} to implement the tasks in the relevant processes.
```
Comment (kaldesai) [inline doc-content/enterprise-only/admin-and-config/manage-service-tasks-proc.adoc:5]: >> Regarding _You can also add custom elements in the designer palette, including name, icon, sub-category, input and output parameters, and documentation._

If the name, icon, sub-category and so on present on the UI maybe you can bold them? wdyt?

**Review by kaldesai — APPROVED** [review]
One minor suggestion :) Otherwise, your changes look good to me. Thank you.

File: doc-content/enterprise-only/admin-and-config/manage-service-tasks-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/admin-and-config/manage-service-tasks-proc.adoc:5]: Good catch. These are not referencing specific UI elements so they are ok not being bolded imo. Thank you, Kalyani!

File: assemblies/assembly-custom-tasks-and-work-item-handlers.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ ifdef::context[:parent-context: {context}]
 // Purpose statement for the assembly
 As a business rules developer, you can create custom tasks and work item handlers to execute custom code within your process flows and extend the operations available for use in {PRODUCT}. You can use custom tasks to develop operations that {PRODUCT} does not directly provide and include them in process diagrams. Custom tasks and work item handlers support {CENTRAL}, standalone editors, and Visual Studio Code (VSCode). This chapter describes how to use {CENTRAL} to create, manage, and deploy custom tasks and work item handlers.
 
-In {CENTRAL}, each task in a process diagram has a `WorkItem` Java class with an associated `WorkItemHandler` Java class. The work item handler contains Java code registered with {CENTRAL} and implements `org.kie.api.runtime.process.WorkItemHandler`.
+In {CENTRAL}, each task in a process diagram has a `WorkItemDefinition` Java class with an associated `WorkItemHandler` Java class. The work item handler contains Java code registered with {CENTRAL} and implements `org.kie.api.runtime.process.WorkItemHandler`.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply assemblies/assembly-custom-tasks-and-work-item-handlers.adoc:10]: "You can define a `WorkItemDefinition` using a MVFLEX Expression Language (MVEL) map as part of a MVEL list in a  `*.wid` file. Place the `*.wid` file in the `global` directory under the project root or in the directory of the BPMN file."

**Pull request merged**